doc/implementation-selection.md

```diff
@@ -81,3 +81,11 @@ can select the CPU architecture yourself:
 // Use the fallback implementation, even though my machine is fast enough for anything
 simdjson::active_implementation = simdjson::available_implementations["fallback"];
 ```
+
+
+Further Reading
```

Comment:
What are these additions about?

Reply:
@jkeiser We can do away with them, but since the reader does not start with README.md when browsing from doxygen,  but rather with basic.md, the reader does not have any hint regarding the organization of the documentation. This is meant to help the reader navigate. Also, it is way to entice readers to keep on reading...

The best way to assess this PR is to generate the doxygen HTML and see what happens.

I think that the way we have it right now is broken. We start with a blank page and a link to the README which exits the user from the doxygen documentation. People have to figure out that they need to use the menu to navigate... and if it is the first exposition to simdjson you have, it is super rough.

I am trying to have something smoother for release 0.4.

Reply:
It should probably have Basics in it, I'd guess, so that people can get back. And Implementation Selection doesn't need to be in this file (since it seems you removed Performance from the performance.md--I don't inherently mind if all the indexes are the same).

At any rate, we can update it over time, this is fine.